<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetNewNet</class>
 <widget class="QDialog" name="SetNewNet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLit">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>6</y>
     <width>50</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Литера:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNN">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>6</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Номер cети:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnBxNN">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>9</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>126</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnBxLit">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>9</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblSkipNum_txt">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>38</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Допустимое
число пропусков:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBxBindPeriod">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>44</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblControlPeriod_txt">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>72</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Период контроля:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnBxSkipNum">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>44</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblBindPeriod_txt">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>46</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Период связи:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblControlPeriod">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>72</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lblControlPeriod</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblSysAddress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Системный адрес:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnBxSysAddress">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetNewNet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetNewNet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
